README.md

```diff
@@ -46,6 +46,10 @@ You can refer our [Code of Conduct](https://github.com/dell/dellemc-openmanage-a
 ## Testing
 See [here](https://github.com/dell/dellemc-openmanage-ansible-modules/tree/collections/tests/README.md) for further information on testing.
 
+## Debugging
+
+See [How to Debug](.github/debug.md)
```

Comment:
probably you can reword it to See how to debug OpenManage Ansible Modules using IDE
```suggestion
To debug OpenManage Ansible Modules using IDE, see [here](.github/debug.md)
```